source/arch/begin_run_event.json

```diff
@@ -0,0 +1,50 @@
+{
+    "properties": {
+        "beamline_config": {
+            "type": ["object", "string"],
+            "description": "Meta-data regrading how the beamline is configured.  Maybe UID into another collection"
+        },
+        "project": {
+            "type": "string",
```

Comment:
How about we call this identifier? Project type might mean different things for different people.